### ISSUE TITLE
XML config is not recreated in mc setup

### ISSUE THREAD

**Issue by tomgross** [collective/collective.recipe.solrinstance#58]
If an element in a MultiCore setup is changed the configuration files are not recreated.
This is counter intuitive to how zc.buildout works normally.

The configuration files _ARE_ recreated if the main part changes.

Version 6.0b3

**Comment by tomgross** [comment]
A workaround is to change the main part by adding / changing a comment, like this:

```
[solr]
recipe = collective.recipe.solrinstance:mc
...
additional-schema-config =
    <!-- Hi -->
```

**Comment by mauritsvanrees** [comment]
I am seeing this too.
If was fixed before in issue #24 but it has returned.

**Comment by mauritsvanrees** [comment]
This was fixed in the merged pull request.
